plutus-tx-plugin/test/Plugin/Basic/Spec.hs

```diff
@@ -63,3 +66,16 @@ strictLetRec = plc (Proxy @"strictLetRec") (\(x::Integer) (y::Integer) -> let !z
 
 ifOpt :: CompiledCode Integer
 ifOpt = plc (Proxy @"ifOpt") (if ((1 `Builtins.divideInteger` 0) `Builtins.equalsInteger` 0) then 1 else 1)
+
+monadicDo :: CompiledCode (Maybe Integer -> Maybe Integer -> Maybe Integer)
```

Comment:
Single  short comment why you added this example? To test if extra error pattern matching happens

Reply:
:+1: